examples/structs/structs.go

```diff
@@ -50,4 +50,15 @@ func main() {
 	// Structs are mutable.
 	sp.age = 51
 	fmt.Println(sp.age)
+
+	// Structs don't always have to be defined as instances of types,
```

Comment:
Thanks for the contribution, it's useful and I'm inclined to accept it.

Let's work out the details: I feel this comment is inaccurate because `dog` is in fact an instance of a type, just a type that isn't named. It should say something more to the tune of unnamed struct type (but also the word `anonymous` should be there.

Reply:
Sorry for the late response. 
Which one do you think is better?

> Structs don't always have to be defined as instances of named types, you can define a struct anonymously.

or

> Structs can be instances of unnamed types, therefore you can define structs anonymously.

or we could simply go with "You can define structs anonymously".